packages/merge-styles/.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
```

Comment:
Ignore this - will remove this & the other vscode settings json file in actual PR.I